src/main/java/mat/dao/clause/CQLLibraryDAO.java

```diff
@@ -46,5 +48,7 @@
 
 	String getLibraryNameIfDraftAlreadyExists(String setId);
 
-	CQLLibrary findByNameAndVersion(String name,double releaseVersion, int revisionNumber);
+    List<CQLLibrary> getDraftLibraryBySet(String cqlSetId);
```

Comment:
something with alignment. do you have any custom intellij settings?

Reply:
Not really and this looks fine in IDE